config/suppressions.xml

```diff
@@ -17,6 +17,9 @@
              files="TokenTypes.java|IndentationCheckTest.java|XdocsPagesTest|CheckerTest.java"
              lines="1"/>
 
+  <!-- 'format' property of 'IllegalTokenText' exceeds column limit -->
+  <suppress id="lineLengthXml" files="src[\\/]main[\\/]resources[\\/]google_checks.xml"/>
```

Comment:
this is too agressive to suppress all violations in this file from this Check

please use special Check instance for it , as suggested https://github.com/checkstyle/checkstyle/pull/17609#discussion_r2269801611 ,
update is expected only in checkstyle_checks.xml , nothing in config/suppressions.xml